apps/pdda/serviceaccount/demo.yaml

```diff
@@ -4,4 +4,4 @@ metadata:
   name: ${NAMESPACE}
   namespace: ${NAMESPACE}
   annotations:
-    azure.workload.identity/client-id: "424b1e06-4b81-4d49-a5b4-ed120e6ff736"
+    azure.workload.identity/client-id: ""
```

Comment:
Should this be empty?

Reply:
yeah we dont have demo, in hindsight i should just delete this

Reply:
Reviewed the deletion.